bears/java/CheckstyleBear.py

```diff
@@ -10,6 +10,13 @@
     "geosoft": "http://geosoft.no/development/geosoft_checks.xml"}
 
 
+def invalid_configuration(checkstyle_configs, use_spaces, indent_size):
+    if checkstyle_configs != 'google':
+        return
```

Comment:
also I think you should research for the other configs on what's invalid for those and include those cases in here